src/component.js

```diff
@@ -136,7 +136,7 @@ const TinyRendererComponentMixin = {
     return this.node;
   },
 
-  receiveComponent() {
+  receiveComponent(nextElement, transaction, context) {
```

Comment:
But ya, our JSON renderer is just a single shot, so there's no real update step. I think it's useful to have the code here for documentation purposes.